junit/src/main/java/net/fabricmc/loader/impl/junit/FabricLoaderLauncherSessionListener.java

```diff
@@ -38,8 +41,12 @@ public FabricLoaderLauncherSessionListener() {
 		final Thread currentThread = Thread.currentThread();
 		final ClassLoader originalClassLoader = currentThread.getContextClassLoader();
 
+		// parse the test environment type, defaults to client
+		final EnvType envType = Optional.ofNullable(System.getProperty(SystemProperties.SIDE))
+				.map(FabricLoaderLauncherSessionListener::parseEnvType).orElse(EnvType.CLIENT);
```

Comment:
```suggestion
		final EnvType envType = EnvType.valueOf(System.getProperty(SystemProperties.SIDE, EnvType.CLIENT.name()).toUpperCase(Locale.ROOT));
```
No need to use Optional here, and I dont think the function is really needed either.